librz/core/cconfig.c

```diff
@@ -2112,17 +2112,17 @@ static bool cb_scrhtml(void *user, void *data) {
 	return true;
 }
 
-static bool cb_newshell(void *user, void *data) {
+static bool cb_rzshell(void *user, void *data) {
 	RzConfigNode *node = (RzConfigNode *)data;
 	RzCore *core = (RzCore *)user;
 	core->use_tree_sitter_rzcmd = node->i_value;
 	return true;
 }
 
-static bool cb_newshell_autocompletion(void *user, void *data) {
+static bool cb_rzshell_autocompletion(void *user, void *data) {
 	RzConfigNode *node = (RzConfigNode *)data;
 	RzCore *core = (RzCore *)user;
-	core->use_newshell_autocompletion = node->i_value;
+	core->use_rzshell_autocompletion = node->i_value;
```

Comment:
use_shell_autocompletion

Reply:
there is already oldshell autocompletion and that's the default for  now. So `use_shell_autocompletion` might be confusing?

Reply:
rename old shell stuff. so it's easier to remove later

Reply:
Agree for the renaming of the oldshell.

Reply:
I think it's easier/faster to do this renaming as we remove the unused code and we'll be left only with the newshell functions. Otherwise now we have to switch all existing uses of some functions to be _oldshell